### ISSUE TITLE
do not use "which"

### ISSUE THREAD

**Issue by stilsch** [robaato/borgbackup-automate#1]
change
if which findmnt &>/dev/null
to
if hash findmnt &>/dev/null

"which" is not available on serveral distros. "hash" is shell-build in.

**Comment by robaato** [comment]
Good catch. In fact there are three alternatives here: hash, type and command (the last one is POSIX compliant).
http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script/677212#677212
